editoast/src/core/mod.rs

```diff
@@ -5,6 +5,7 @@ pub mod infra_state;
 pub mod mocking;
 pub mod pathfinding;
 pub mod simulation;
+pub mod stdcm;
```

Comment:
Is this needed anymore ?

Reply:
Why would the module *declaration* be unneeded ?